### PR TITLE
feat: safe connector

### PR DESCRIPTION
**Pull request by DanielSinclair**
## Changes:
- introduced `safeWallet` that relies on new `SafeConnector` in Wagmi
- added `safeWallet` to default wallet list
- Bumped Wagmi to `11.0.6`
- Added Safe App `manifest.json` and icons/favicons to `packages/example`

### CODE REVIEW

File: packages/rainbowkit/src/wallets/walletConnectors/safeWallet/safeWallet.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+
+export interface SafeWalletOptions {
+  chains: Chain[];
+  options?: SafeConnectorOptions;
+}
+
+// See the official integration guide for auto-connecting
+// https://github.com/safe-global/safe-apps-sdk/tree/main/packages/safe-apps-wagmi#integration-steps
```
Comment (ribeirojose) [inline packages/rainbowkit/src/wallets/walletConnectors/safeWallet/safeWallet.ts:12]: 🙇 thanks for leaving this reference to the docs here

**Comment by maxwell-1999** [comment]
Guyz please review and merge it fast.
I need Safe Wallet connector ASAP

**Comment by dkrish** [comment]
would love to be able to support this on SuperRare, hope it gets approved soon!

**Review by ribeirojose — APPROVED** [review]
Not sure whether I could approve this, but looking good to me! Also tested the example app in the Safe's iFrame context and could connect/send tx without any issues.

**Comment by ConnorSB13** [comment]
We need it from Enso, pls approve!

**Review by nickbytes — APPROVED** [review]

**Comment by DanielSinclair** [comment]
Now available as [v0.12.4](https://github.com/rainbow-me/rainbowkit/releases/tag/%40rainbow-me%2Frainbowkit%400.12.4)

**Comment by dkrish** [comment]
@DanielSinclair unfortunately, not seeing it in the list of wallets once we import the connector in our component (after updating to v0.12.4)

**Comment by DanielSinclair** [comment]
@dkrish Will automatically connect, and Safe will be visible in the wallet list in case a user disconnects. Did you test this in the Safe Wallet browser?

**Comment by SamuraiT** [comment]
soooooo cool

**Pull request merged**